### PR TITLE
Add prometheus config example

### PR DESCRIPTION
**Pull request by Goala**

### CODE REVIEW

File: local-tests/prometheus/prometheus.yml [hunk 1]
```diff
@@ -6,3 +6,14 @@ scrape_configs:
...
     static_configs:
       - targets: ['localhost:9090']
+
+  - job_name: 'amp'
+    metrics_path: '/amp/metrics'
+    params:
+      sap-client: [001]
+      scenario: [example-prom]
+    basic_auth:
+      username: 'DEVELOPER'
```
Comment (JohannesKonings) [inline local-tests/prometheus/prometheus.yml:16]: Could you change the credentials to env variables like in the telegraf config and add a note in the readme?

**Review by JohannesKonings — CHANGES_REQUESTED** [review]

File: local-tests/prometheus/prometheus.yml (hunk 1 above)
Comment (Goala) [reply local-tests/prometheus/prometheus.yml]: @JohannesKonings it looks like prometheus does not support env vars für prometheus.yml
https://stackoverflow.com/questions/50009932/environment-variables-in-prometheus-values

I removed the credentials with placeholders

Comment (JohannesKonings) [reply local-tests/prometheus/prometheus.yml]: OK, we should try to document this somwhere

**Review by JohannesKonings — APPROVED** [review]
LGTM

**Pull request merged**